### PR TITLE
Abhishek_Assignment_19_(40+10+10+10+15)Minutes

### PR DESCRIPTION
**Pull request by AbhishekSJune2020**
Output1:
Count of Prime Number between range :25
Sum of Prime Number between range :1060
Average of Prime Number between range 2 - 100 is: 42.0

Output:2
Values are not matching at index -> 2
 
Output: 3
Values are not matching at index -> 2
Values are not matching at index -> 4		

Output-4
Values are not matching at index -> 2, From (9,18) max value is18

Output:5
Values are not matching at index -> 2, [arr1 value is Krishna, arr2 value is Technocredits]
Values are not matching at index -> 3, [arr1 value is Technocredits, arr2 value is Krishna]

### CODE REVIEW

File: src/abhishekSJun20/FirstDifferIndexOfArray.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+public class FirstDifferIndexOfArray {
+
+	void differIndexValueInIndex(int[] arr1, int[] arr2) {
+
+		for (int index = 0; index < arr1.length; index++) {
+			if (arr1[index] != arr2[index])
+				System.out.println("Values are not matching at index -> " + index);
+		}
```
Comment (KrishnaKTechnocredits) [inline src/abhishekSJun20/FirstDifferIndexOfArray.java:16]: Here you are checking for the first index, why return statement is missing ??

File: src/abhishekSJun20/PrimeNumber.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+		// Method displays and return sum of all the prime number in the range
+		int sum = 0;
+		for (int index = startRange; index <= endRange; index++) {
+			sum += primeNumber(index);
+		}
+		System.out.println("Sum of Prime Number between range :" + sum);
+		return sum;
+	}
+
+	double AverageAllPrimeNumInRange(int sumOfPrimeNum, int countOfPrimeNum) {
```
Comment (KrishnaKTechnocredits) [inline src/abhishekSJun20/PrimeNumber.java:50]: No need to write extra methods, Just create two instance variable sum and count and perform that operation in isPrime method and use those variable in method like:
void findPrimeNumberInGivenRange(int startNumber, int endNumber) {
for(int index=startNumber; index<=endNumber; index++) {
verifyNumberIsPrimeOrNot2(index);
}
System.out.println("Total count is: "+count);
System.out.println("Sum of all Prime number is: "+sum);
System.out.println("Average of all prime number is: "+sum/count);

**Review by KrishnaKTechnocredits — CHANGES_REQUESTED** [review]
.

File: src/abhishekSJun20/PrimeNumber.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+	int primeNumber(int num) {
+		// Method to find if Number is Prime Number or not. It return num - If prime and
+		// returns 0 - If not Prime
+		for (int index = 2; index <= Math.sqrt(num); index++) {
+			if (num % index == 0) {
+				return 0;
+			}
+		}
+		return num;
```
Comment (KrishnaKTechnocredits) [inline src/abhishekSJun20/PrimeNumber.java:22]: Where is the printing statement, number is prime or not?

File: src/abhishekSJun20/FirstDifferIndexOfArray.java (hunk 1 above)
Comment (AbhishekSJune2020) [reply src/abhishekSJun20/FirstDifferIndexOfArray.java:16]: done changes as per comment.

File: src/abhishekSJun20/PrimeNumber.java (hunk 3 above)
Comment (AbhishekSJune2020) [reply src/abhishekSJun20/PrimeNumber.java:22]: done changes as per comment

**Pull request merged**